### PR TITLE
Add a project tag to builds for filtering (#2263).

### PR DESCRIPTION
**Pull request by oliverchang**

### CODE REVIEW

**Comment by oliverchang** [comment]
TBR to get ball rolling since we need to kick off new builds.

File: infra/gcb/build_project.py [hunk 1]
```diff
@@ -355,7 +355,10 @@ def run_build(build_steps, tag):
       'timeout': str(BUILD_TIMEOUT) + 's',
       'options': options,
       'logsBucket': GCB_LOGS_BUCKET,
-      'tags': [ tag ],
```
Comment (inferno-chromium) [inline infra/gcb/build_project.py]: Does this need to go in tags field. In original patch, i see it in 

'images': [ image ],

**Comment by oliverchang** [comment]
thanks! will upload a follow up PR to completely fix this, since that's reliant on new builds being available first.

**Comment by Dor1s** [comment]
Looks like this broke code coverage builder, fixed in 497cd398d0798bc834506880921c35919790782c

**Comment by oliverchang** [comment]
thanks!

**Pull request merged**